f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -1290,6 +1307,7 @@ def _update_member_status(self, members):
                             plugin_const.ACTIVE,
                             lb_const.ONLINE
                         )
+                        member['provisioning_status'] = plugin_const.ACTIVE
```

Comment:
This doesn't seem correct to force the member status to active.  What if the member's subnet is missing a seg-id and is the thing requiring service resync?  Maybe this works because if the service is complete the code will not even call _update_member_status.